### DISCUSSION TITLE
Is the a way to update the dataset periodically?

### DISCUSSION THREAD

**Discussion by mrtushartiwari** [huggingface/datasets#5808] [General]
There are some dataset that requires periodic updates like Interest Rates by the central banks, production of power by plant etc.

Even for eg like E commerce reviews that gets added over time.

Is there a way to update/ add new version of dataset available?

**Comment by mariosasko** [comment]
You can update a dataset by uploading a data file that has the same format as the existing data files in the repo (to work with our [data files resolution](https://huggingface.co/docs/datasets/repository_structure)). 

And you can use Git tags for versioning ([`huggingface_hub.create_tag`](https://huggingface.co/docs/huggingface_hub/v0.14.1/en/package_reference/hf_api#huggingface_hub.HfApi.create_tag)). These tags can then be referenced via the `revision` parameter in `load_dataset`:
```python
ds = load_dataset(`namespace/repo_id`, revision="tag")
```
